webapi/templates/ticket-search-result.html

```diff
@@ -141,6 +143,40 @@ <h1>Vote Choices</h1>
             </tr>
         </table>
 
+        <h1>Alternate Signing Address Data</h1>
```

Comment:
As you have added this new section dedicated for `Alternate Signing Address Data`, I think we can also put the address itself here, rather than in the Ticket section above.